src/components/organisms/FileTreePane/FileTreePane.tsx

```diff
@@ -372,6 +375,8 @@ const TreeItem: React.FC<TreeItemProps> = props => {
     ? fileMap[ROOT_FILE_ENTRY].filePath
     : path.join(fileMap[ROOT_FILE_ENTRY].filePath, treeKey);
 
+  const targetFolder = isRoot ? ROOT_FILE_ENTRY : treeKey.replace(path.sep, '');
```

Comment:
hm.. I think targetFolder should be relative to root - or empty for root!? Does this work for nested folders?

Reply:
It creates files in the nested folders but can't be expanded. (might not be related)

I followed the structure of the folder dropdown in NewResourceWizard.